README.md

```diff
@@ -178,3 +178,37 @@ $ oc adm release new --from-release=registry.svc.ci.openshift.org/openshift/orig
 $ cd ../installer
 $ OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=docker.io/sttts/origin-release:latest bin/openshift-install cluster ...
 ```
+
+## Tests
+
+This repository uses the [OpenShift Tests Extension (OTE)](https://github.com/openshift-eng/openshift-tests-extension) framework for end-to-end tests.
+
+### Building the test binary
+
+```bash
+make build
+```
+
+### Running test suites and tests
```

Comment:
at some point we could move logic to the `build-machinery-go` repository for easier execution of tests.